corehq/apps/settings/views.py

```diff
@@ -264,6 +266,9 @@ def post(self, request, *args, **kwargs):
             try:
                 self.request.couch_user.delete_domain_membership(self.domain_to_remove, create_record=True)
                 self.request.couch_user.save()
+                log_user_change(request.domain, couch_user=request.couch_user,
+                                changed_by_user=request.couch_user, changed_via=USER_CHANGE_VIA_WEB,
+                                message=_("Removed from domain"))
```

Comment:
should this include the domain that the user was removed from?

Reply:
Oh yes, absolutely. Missed that the domain on the User History record would not necessarily be the domain that is being removed. Will add that here and also at other places where and if needed in other PRs.

Reply:
https://github.com/dimagi/commcare-hq/pull/29973/commits/fb324eb0ffd3ce5272ea245d16f135824d831ff0